src/client/app/redux/selectors/uiSelectors.ts

```diff
@@ -409,9 +409,12 @@ export function getSelectOptionsByEntity(
 				defaultGraphicUnit: defaultGraphicUnit
 			} as SelectOption;
 		});
-
-	const compatible = sortBy(compatibleItemOptions, item => item.label.toLowerCase(), 'asc');
-	const incompatible = sortBy(incompatibleItemOptions, item => item.label.toLowerCase(), 'asc');
+	// TODO: change second argument of locale from undefined to current language if needed, but alphabetical ordering works
```

Comment:
From my looking, this is called by two functions in src/client/app/redux/selectors/uiSelectors.ts where both are createAppSelector. In selectPossibleMeterUnits() in adminSelectors.ts that is also a createAppSelector you added to access the selected language. Thus, I'm wondering if it is possible to do the same in the uiSelectors functions and then pass the language to this function for use. What do you think?

Reply:
I'll look into it right now.

Reply:
Finished implementation of changes.